exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/metadata/MetadataBase.java

```diff
@@ -108,6 +105,18 @@
 
 
   public static abstract class ColumnMetadata {
+
+    /**
+     * Number of nulls is considered to be valid if its value is not null and -1.
+     *
+     * @return true if nulls value is defined, false otherwise
+     */
+    @JsonIgnore
+    public boolean isNullsDefined() {
+      Long nulls = getNulls();
```

Comment:
Why is it necessary to deal both with `null` and `-1`? Why is `getNulls` a `Long` and not a `long`? Consider renaming it to `getNumNulls`.

Reply:
Previously created Drill metadata files might have nulls defined as null. Agree that renaming is good idea but since it will affect several versions of `ColumnMetadata` class which is used in ser /de when creating Drill parquet metadata files, I'd rather not make such changes right now. Since PR resolves release blocker, I suggest we keep changes at minimum to avoid possible regressions. Created Jira for the improvement - https://issues.apache.org/jira/browse/DRILL-6620.

Reply:
@arina-ielchiieva My major concern is not the name of the method, but using `Long` when `long` should be used. Using `long` will also address my second major concern: using `null` and `-1` for "not set" definition.

Reply:
I know and totally agree that `long` usage is more correct choice here but this was done before and I left `Long` for backward compatibility plus did not want to change all `ColumnMetadata` successors yo avoid any problems with deserialization. I think we can make this changes in subsequent Jira DRILL-6620, added comment in Jira.